MPM/mpm_settings.php

```diff
@@ -576,4 +576,33 @@ public function add_notice ($message, $type = 'notice')
 			}
 		}
 	}
+
+	/**
+	 * Restocks an order, a failed or cancelled order for example
+	 *
+	 * @param WC_Order $order
+	 */
+	public function restock_order(WC_Order $order)
```

Comment:
Laat de `reduce_sales` maar zitten voor nu. Wanneer je namelijk een `on-hold` bestelling handmatig in de admin op `cancelled` of `failed` zet worden deze sales ook niet afgeboekt.